bin/weewx/drivers/vantage.py

```diff
@@ -2044,6 +2044,11 @@ def add_options(self, parser):
         parser.add_option("--dump", action="store_true",
                           help="Dump all data to the archive. "
                                "NB: This may result in many duplicate primary key errors.")
+        parser.add_option("--dump-quick", action="store_true", dest="dump_quick",
+                          help="Dump all data to the archive, fetching all data off the console"
```

Comment:
You need a space at the end of these lines. Otherwise the `--help` command will merge the two words. 

Try it.

Reply:
Indeed, I didn't even catch that when I ran my tests. Both requests merged!